core/src/main/java/com/cryptape/cita/protocol/system/CITAjSystemContract.java

```diff
@@ -116,72 +116,72 @@ public long getQuotaPrice(String from) throws IOException {
     }
 
     public boolean approveNode(
-            String nodeAddr, String adminPrivatekey, int version, BigInteger chainId)
+            String nodeAddr, String adminPrivateKet, int version, BigInteger chainId)
```

Comment:
`adminPrivateKet` is a typo, should be `adminPrivateKey`

Reply:
I changed